assets/js/storage.js

```diff
@@ -1,6 +1,8 @@
 angular
 .module('storage', [])
 .factory('$storage', function() {
+  var likedVideos = getObj('likes');
```

Comment:
I don't think you would have this problem if you left it as a function that returns the value